<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeywordTabWidget</class>
 <widget class="QWidget" name="KeywordTabWidget">
  <layout class="QVBoxLayout" name="verticalLayout_keyword">
   <item>
    <widget class="QLabel" name="keywordLabel">
     <property name="text">
      <string>&lt;h2&gt;关键词管理模块&lt;/h2&gt;&lt;p&gt;在此处实现词库的添加、编辑、删除等功能。&lt;/p&gt;&lt;p&gt;当前模块独立于主窗口逻辑，易于维护！&lt;/p&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_category_select">
     <item>
      <widget class="QLabel" name="label_category">
       <property name="text">
        <string>选择分类:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="keywordCategorySelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_current_keywords">
     <property name="text">
      <string>当前关键词列表 (系统/自定义合并):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="currentKeywordsText">
     <property name="maximumHeight">
      <number>100</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#888888;&quot;&gt;选择分类后将在此处展示当前分类下的所有关键词...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_edit_keywords">
     <property name="text">
      <string>编辑自定义关键词 (逗号或换行分隔，将覆盖该分类的自定义关键词):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="keywordInput">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="placeholderText">
      <string>输入自定义关键词，例如：私盐, 走私, 缉捕...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveKeywordsButton">
     <property name="styleSheet">
      <string notr="true">background-color: #2d5a3e; color: white; padding: 10px;</string>
     </property>
     <property name="text">
      <string>保存关键词</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_keyword_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>